app/javascript/angular/code/controllers/_fixed_sessions_map_ctrl.js

```diff
@@ -94,4 +96,61 @@ export const FixedSessionsMapCtrl = (
     }, true);
 
   $scope.setDefaults();
+
+  if (process.env.NODE_ENV !== 'test') {
```

Comment:
We could consider extracting the same parts into an abstraction but I'm not sure it wouldn't be wasted effort. Especially cause there's already the mobile / fixed duplication. Let's just not forget. If you want on my wall there's an "eject" section where I was collecting things to be done before wrapping up